src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc

```diff
@@ -0,0 +1,2125 @@
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIDeviceStatusNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIDriverBrakingNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIEngineTorqueNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIExternalTemperatureNotification>(
+            (*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIFuelLevelNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIFuelLevelStateNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIGpsDataNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIHeadLampStatusNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIInstantFuelConsumptionNotification>(
+            (*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIMyKeyNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIOdometerNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIPrndlNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIRpmNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVISpeedNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVISteeringWheelAngleNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVITirePressureNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIVehicleDataNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIVinNotification>((*it)));
+    ++it;
+    commands_list_manage_mobile_.push_back(
+        CreateCommand<am::commands::OnVIWiperStatusNotification>((*it)));
+    ++it;
+  }
+
+  void FillMessagesListCommandsToHmi() {
+    for (uint32_t i = 0; i < kCountCommandsToHmi_; ++i) {
+      messages_list_for_commands_to_hmi_.push_back(CreateMessage());
+    }
+  }
+
+  void FillCommandsListToHmi() {
+    std::vector<MessageSharedPtr>::iterator it =
+        messages_list_for_commands_to_hmi_.begin();
+
+    commands_list_to_hmi_.push_back(
+        CreateCommand<am::commands::OnAppPermissionChangedNotification>((*it)));
+    ++it;
```

Comment:
@okozlovlux These three lines are repetitive with only one argument changing. Perhaps a macro or template will be useful here.